staging/src/k8s.io/code-generator/cmd/conversion-gen/main.go

```diff
@@ -14,20 +14,38 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// conversion-gen is a tool for auto-generating Conversion functions.
+// conversion-gen is a tool for auto-generating functions that convert
+// between internal and external types.  A general conversion task
```

Comment:
I think "conversion code generation task" is more clear than "conversion task".

Reply:
Yes, that's better.  Updated.